templates/app/index.html

```diff
@@ -6,7 +6,8 @@
     <title><%= name %></title>
     <meta name="description" content="<%= name %> : <%= description %>">
     <link rel="stylesheet" type="text/css" media="screen" href="styles/stylesheet.css">
-    <script src="scripts/offline-manager.js"></script>
+    <script data-worker="offliner-worker.js" src="lib/offliner/dist/offliner-client.min.js"></script>
+    <script defer src="scripts/index.js"></script>
```

Comment:
@mykmelez, answering each item:

  1. You're right. I will recover the original name.
  2. I think is better to hide the details. I will rename it to the original name as well.
  3. Well this is tricky. offliner is a library and it provides a way to communicate with the worker you design (in this case, the worker oghliner automatically writes). This is the client. [It is intended to be controlled by client code](https://github.com/mozilla/oghliner/pull/96/files#diff-6d5cc80376a3a8d696a6bd33cbbc2a9eR25), this is why we use two scripts: one is the library, the other is your code. I have two proposals to solve this:
    1. Add a special `data-autoinstall="true"` to offliner to make it simply run.
    2. Lazy-load the library in the client code before installing.

What do you think?